src/qt/guiconstants.h

```diff
@@ -56,7 +56,7 @@ static const int TOOLTIP_WRAP_THRESHOLD = 80;
 static const int MAX_URI_LENGTH = 255;
 
 /* QRCodeDialog -- size of exported QR Code image */
-#define EXPORT_IMAGE_SIZE 256
+#define QR_IMAGE_SIZE 350
```

Comment:
why this change in image size?

Reply:
it was a carry over from the ABC PR...i think it's fine and actually a bigger image is i think better.  I tried scanning the image from my computer with my cell and it works fine whereas in the past I sometimes had issues (if my memory servers), because of the same image.

Reply:
In ABC, the text representation of the address is embedded in the QR code. The image needed to be increased to fit the text.